client/web/compose/src/components/PageBlocks/Navigation/Configurator.vue

```diff
@@ -0,0 +1,300 @@
+<template>
+  <div>
+    <b-tab :title="$t('navigation.label')">
+      <div class="mb-3">
+        <h5 class="text-primary">
+          {{ $t("navigation.displayOptions") }}
+        </h5>
+
+        <b-row class="justify-content-between">
+          <b-col
+            cols="6"
```

Comment:
a bit more responsive please, cols=12 is always a safe bet, then md the final layout. You can start from there